package.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "discourse-antivirus",
+  "version": "1.0.0",
```

It's not like we actually use it, but… 😛 

https://github.com/discourse/discourse-antivirus/blob/6c9daff9a19d5653749f0336be9041a182ed0e9f/plugin.rb#L5

```suggestion
  "version": "0.1.0",
```